src/pages/account/logins/[type]/[id].tsx

```diff
@@ -40,34 +42,23 @@ const IdentityOverviewPage: React.FC<{ identity: Identity }> = ({
 
 export default IdentityOverviewPage;
 
-export const getServerSideProps: GetServerSideProps = withSSRLogger(
-  withSession(async (context) => {
-    try {
-      const userDocumentId = context.req.session.get("user-session-id");
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  return wrapMiddlewaresForSSR<{ type: string }>(
+    context,
+    [withLogger, withSentry, withMongoDB, withSession, withAuth],
+    async (request: ExtendedRequest, response: ServerResponse) => {
+      if (!context?.params?.id) {
+        response.statusCode = 400;
+        response.end();
+        return;
```

Comment:
Any reason why you did not went with `return response.end()`?

Reply:
psychological safety, I wanted to be sure to have a function that return void at the moment, and maybe let the possibility of doing something else (but not really no, to answer honestly)

Reply:
> psychological safety
I had the feeling you did it for this, and I'm more than fine with it.